blps/blp-703-servies.md

```diff
@@ -11,29 +11,35 @@
 ## Types
 - `Redirection` would be the most trivial one
     - it will require a regex based replacement to the service endpoint
-- Down the road we will come up with more types
+- Down the road we will come up with more types, e.g. `Source` or `Binary`
```

Comment:
Isn't Source already implemented? The example below shows it. Why is it only "down the road"?